### PR TITLE
[da] Do not use title case

### PR DESCRIPTION
**Pull request by c960657**
Danish does not use Title Case for in headlines and definitely not in expressions like “128 Bytes” or “5,4 Millioner”. Apart from actual movie/book titles, Title Case is almost only used with proper nouns, certain titles and pronouns and in abbreviations.

The authoritative rules for letter case in Danish are described here:
https://sproget.dk/raad-og-regler/Retskrivningsregler/retskrivningsregler/a7-11-14

### CODE REVIEW

**Comment by pama** [comment]
@c960657 I noticed a change from KB to kB, but the other units are uppercase.

File: rails/locale/da.yml [hunk 1]
```diff
@@ -194,11 +194,12 @@ da:
         format: "%n %u"
         units:
           byte:
-            one: Byte
-            other: Bytes
+            one: byte
+            other: bytes
           gb: GB
-          kb: KB
+          kb: kB
```
Comment (sunny) [inline rails/locale/da.yml:200]:
```suggestion
          kb: KB
```

**Review by sunny — COMMENTED** [review]
Can you also add a CHANGELOG entry? 🙏🏻

**Comment by c960657** [comment]
I intentionally used lowercase “k”, the [SI prefix for “kilo”](https://en.wikipedia.org/wiki/Kilobyte).

The [Danish Orthographical Dictionary](https://en.wikipedia.org/wiki/Retskrivningsordbogen), the authoritative source for Danish spelling, explicitly mentions that [“kilobyte” is abbreviated “kB”](https://dsn.dk/soegning/?soegeord=kilobyte).

The Danish orthography is not always in sync with scientific standards (e.g. “deciliter” is abbreviated “dl” according to the Danish Language Council, but “dL” according to SI), but with respect to “kilobyte” they agree.

**Pull request merged**